### ISSUE TITLE
Extension manager doesn't work when built from source

### ISSUE THREAD

**Issue by antonio-rojas** [jupyterlab/jupyterlab#6303]
When building jupyterlab 0.35.5 from source, the extension manager throws an error: 
`SyntaxError: Unexpected token < in JSON at position 0`
It works fine when installing from pip. After comparing the intall trees in both, I found it to be caused by the different jupyter-lab scripts. When installing from source it's 
```
#!/usr/bin/python
# EASY-INSTALL-ENTRY-SCRIPT: 'jupyterlab==0.35.5','console_scripts','jupyter-lab'
__requires__ = 'jupyterlab==0.35.5'
import re
import sys
from pkg_resources import load_entry_point

if __name__ == '__main__':
sys.argv[0] = re.sub(r'(-script\.pyw?|\.exe)?$', '', sys.argv[0])
sys.exit(
load_entry_point('jupyterlab==0.35.5', 'console_scripts', 'jupyter-lab')()
)
```
while the one from pip is
```
#!/usr/bin/python
# -*- coding: utf-8 -*-
import re
import sys

from jupyterlab.labapp import main

if __name__ == '__main__':
sys.argv[0] = re.sub(r'(-script\.pyw?|\.exe)?$', '', sys.argv[0])
sys.exit(main())
```

Replacing the script with the pip one fixes the problem.

**Comment by vidartf** [comment]
When you say "install from source" which command exactly are you using to install from source?

**Comment by antonio-rojas** [comment]
python setup.py build/install. See full build script here:

https://git.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/jupyterlab

**Comment by vidartf** [comment]
Thanks, just asking for clarity, as pip can install from source too. I don't think I have a nuanced enough understanding of what `python setup.py install` does to try to debug this.

**Comment by jasongrout** [comment]
Can you try in the latest prerelease?

**Comment by antonio-rojas** [comment]
Same with 1.0.0a3

**Comment by antonio-rojas** [comment]
Fixed in 1.0.0 final
